laser_encoders/requirements.txt

```diff
@@ -1,8 +1,8 @@
 fairseq==0.12.2
```

Comment:
I suggest we drop this file, because it is not used anyway: when pip-installing `laser_encoders`, the list of dependencies is pulled from `pyproject.toml` instead.

Reply:
I've just made the requested changes.